src/css/support.js

```diff
@@ -18,6 +18,11 @@ define( [
 			return;
 		}
 
+		// Don't run until window is visible (https://github.com/jquery/jquery-ui/issues/2176)
+		if ( documentElement.offsetHeight === 0 ) {
```

Comment:
Do you have any suggestions? I'm not sure what the rational for setting `height: 0` on the root element would be.

Maybe I can have it check if the width is zero too. I didn't test that yet. I wanted to get your thoughts on that first.

Thanks.

Reply:
I would rather check against `div` directly than against its containing document and assuming accuracy and correspondence. Specifically, I think `div.offsetWidth` would be a good target: https://jsfiddle.net/tjnvs51o/

Reply:
I updated to `div.offsetWidth`. It think it works, though I'm struggling to figure out how to run all the tests.

Reply:
There are a couple ways to run tests:

1. With npm scripts: `npm run browser`. This will run tests on the command line using karma and headless browsers. Set the browser you want with the `BROWSERS` env var. See Gruntfile.cjs.
2. Open the QUnit test page directly and run the test using the QUnit UI. Run `node test/server.js` and visit http://localhost:3000/test/. Your test is in the `support` module so select that from the module list to filter the tests.

Reply:
The test in this PR is failing in Safari. Also, we want to remove the test `container` element before the early return.

Reply:
@timmywil we should document this in README.

Reply:
I'm planning on documenting all of the upcoming test changes (to drop grunt, karma, etc). `node test/server.js` will likely be added to an npm script.